### PR TITLE
Python 3 compatibility fixes

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

**Comment by pypingou** [comment]
I wanted to work on this :(

**Comment by scop** [comment]
Don't worry, I think there's more work for you in this area left after this PR :)

File: pkgdb2client/__init__.py [hunk 1]
```diff
@@ -95,7 +93,7 @@ def ask_password(username=None, bad_password=False):
             username = fedora_cert.read_user_cert()
         except fedora_cert.fedora_cert_error:
             LOG.debug('Could not read Fedora cert, asking for username')
-            username = raw_input("Username: ")
+            username = input("Username: ")
```
Comment (pypingou) [inline pkgdb2client/__init__.py]: When testing this on py2 it seems to not behave as I want:
````
>>> t = input('foo:')
foo:test
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "<string>", line 1, in <module>
NameError: name 'test' is not defined
>>> t = raw_input('foo:')
foo:test
````
But indeed on py3 it's the opposite:
````
>>> t = raw_input('foo:')
Traceback (most recent call last):
  File "<input>", line 1, in <module>
NameError: name 'raw_input' is not defined
>>> t = input('foo:')
foo:test
````

Comment (pypingou) [inline pkgdb2client/__init__.py]: Ok, I just saw the ``from six.moves import input``, that fixes this, cool :)

File: setup.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
         ]
     },
     install_requires=[
-        'requests', 'python-bugzilla', 'python-fedora', 'setuptools',
+        'requests', 'python-bugzilla', 'python-fedora', 'setuptools', 'six',
```
Comment (pypingou) [inline setup.py]: Do we have a minimum version required for six?

**Comment by pypingou** [comment]
This looks all good to me, maybe just the minimal version required for six if there is one.

**Comment by scop** [comment]
I don't know the minimal six version required. I'm using 1.9.0 on F-23.

**Comment by pypingou** [comment]
el7 also has 1.9.0, cool :)

**Pull request merged**